src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -490,22 +490,21 @@ Blank()
 true
 
 >> Text(ParseJSON("1.234"), "#.##")
-"1.23"
+Errors: Error 0-32: The function 'Text' has some invalid arguments.|Warning 25-31: Not supported format for non numeric and datetime in 'Text'.
```

Comment:
That's an interesting decision we need to make. We have done work to treat untyped objects (that hold numbers) the same as numbers themselves, so we should consider enabling it (at compile time) for numbers, and returning a runtime error if a text/boolean is passed.

Reply:
add check for untyped object as well and revert this file change.